vitest.config.ts

```diff
@@ -14,7 +14,11 @@ export default defineConfig({
       './playground-temp/**/*.*',
     ],
     testTimeout: 20000,
-    isolate: false,
+    // isolate: false,
+    // TODO:
+    // importing non entry file can be broken due to cyclic import e.g.
+    //   pnpm exec tsx packages/vite/src/node/server/index.ts
+    setupFiles: ['./packages/vite/src/node/index.ts'],
```

Comment:
Interestingly, export hoisting can make cyclic import handling stricter as seen by:

```sh
$ pnpm test-unit packages/vite/src/node/ssr/runtime/__tests__/server-runtime.spec.ts 

FAIL  packages/vite/src/node/ssr/runtime/__tests__/server-runtime.spec.ts [ packages/vite/src/node/ssr/runtime/__tests__/server-runtime.spec.ts ]
ReferenceError: Cannot access 'serverConfigDefaults' before initialization
 ❯ Module.get [as serverConfigDefaults] packages/vite/src/node/server/index.ts:4:116
```

but this case might be legitimate since this is the same error as `tsx`:

```js
$ pnpm exec tsx packages/vite/src/node/server/index.ts
/home/hiroshi/code/others/vite/packages/vite/src/node/config.ts:648
  server: serverConfigDefaults,
          ^


ReferenceError: Cannot access 'serverConfigDefaults' before initialization
```

Just temporarily, I added silly `setupFiles`, so that modules will be initialized in a known safe order.